tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -84,7 +84,6 @@ var _ = Describe("odo devfile catalog command tests", func() {
 				"java-springboot",
 				"java-quarkus",
 			})
-			helper.CmdShouldPass("odo", "registry", "delete", "fake", "-f")
```

Comment:
Test script is creating the fake registry, so script should clean it up. Please add it back.

Reply:
Done